addins/Cake.NDepend.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.NDepend
 NuGet: Cake.NDepend
-Prerelease: false
 Assemblies:
 - "/**/Cake.NDepend.dll"
-Repository: https://github.com/cake-contrib/cake-ndepend
+Repository: https://github.com/joaoasrosa/cake-ndepend
```

Comment:
@Jericho correct repos seems to now be https://github.com/cake-contrib/cake-ndepend

Reply:
The issue seems to be the repo name `cake-ndepend` under the `cake-contrib` user does not match the addin name which is `Cake.NDepend`. The author has not updated his nuspec and the audit tool is unable to figure out that this addin was moved to the cake-contrib organization due to the different spelling.